pandas/core/algorithms.py

```diff
@@ -1462,7 +1462,7 @@ def take(
 
     Examples
     --------
-    >>> from pandas.api.extensions import take
+    >>> from pandas.pai.extensions import take
```

Comment:
this is a typo. are we testing these doc-tests?

Reply:
No, I think we're only showcasing the functionality of the take method by passing in different keywords.